src/Model/Cli/Runner/Drush.php

```diff
@@ -249,9 +380,12 @@ private function ensureLocalSettings()
         $finder = new Finder();
         $finder->files();
         $finder->in(getcwd());
-        $finder->name('settings.local.php');
+        $filename = $this->config->getLocalSettingsFilename();
```

Comment:
What is the motivation behind this change? Is this to distinguish between sites that are older that may have a `secret.settings.php` file?

Reply:
I had a project where the local settings file was named something differently. This lets you define the secret/local settings file's filename per project.